src/Views/OperatingSystemView.vala

```diff
@@ -713,4 +731,60 @@ public class About.OperatingSystemView : Gtk.Box {
             add_css_class ("link");
         }
     }
+
+    private class SponsorUsRow : Gtk.ListBoxRow {
+        public string uri { get; construct; }
+
+        public SponsorUsRow (string uri) {
+            Object (
+                uri: uri
+            );
+        }
+
+        class construct {
+            set_accessible_role (LINK);
+        }
+
+        construct {
+            var image = new Gtk.Image.from_icon_name ("face-heart-symbolic") {
+                pixel_size = 16
+            };
+            image.add_css_class (Granite.STYLE_CLASS_ACCENT);
+            image.add_css_class ("pink");
+
+            var labels_stack = new Gtk.Stack () {
+                hexpand = true,
+                transition_type = SLIDE_UP_DOWN
+            };
+            labels_stack.add_named (new Gtk.Label (_("Sponsor Us")) {
+                halign = START
+            }, "title");
+            labels_stack.add_named (new Gtk.Label (_("23% towards $7,500 per month goal")) {
+                halign = START
+            }, "goal");
+
+            var progress_bar = new Gtk.ProgressBar ();
```

Comment:
Should probably use a LevelBar so you don't get the little glowy progress animation